.github/workflows/native-docker-labeled/native_docker_labeled.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import argparse
+import requests
+import os
+import sys
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Deploy a pack from a contribution PR to a branch')
+    parser.add_argument('-c', '--changed_files', help='list of the changed files')
+    args = parser.parse_args()
+    changed_files = args.changed_files.split(" ")
+    return validate_native_docker(changed_files)
+
+
+def validate_native_docker(changed_files):
+    """
+    This function validate that no native docker supported dockers are being updated.
+    Args:
+        changed_files: the list of files that are being changed in the current pr.
+
+    Returns: the exit code according to wether the list of native docker supported dockers is empty or not.
+    """
+    updated_supported_dockers = get_updated_supported_dockers(changed_files)
+    if updated_supported_dockers:
+        print(f"the following dockers are updated and supported by the native docker: {', '.join(updated_supported_dockers)}. Please make sure to update the native docker accordingly and add the 'native image approved' label.")
```

Comment:
We should refer them to the docs here. Like:
Check what is required to meet the criteria for this label [here](example.com) and then add the "native image approved" label.